paginator/paginator_test.go

```diff
@@ -24,10 +24,11 @@ func TestPaginator(t *testing.T) {
 /* models */
 
 type order struct {
-	ID        int       `gorm:"primaryKey"`
-	Remark    *string   `gorm:"type:varchar(30)"`
-	CreatedAt time.Time `gorm:"type:timestamp;not null"`
-	Data      JSON      `gorm:"type:jsonb"`
+	ID          int        `gorm:"primaryKey"`
+	Remark      *string    `gorm:"type:varchar(30)"`
+	CreatedAt   time.Time  `gorm:"type:timestamp;not null"`
+	Data        JSON       `gorm:"type:jsonb"`
+	Description NullString `gorm:"type:varchar(30)"`
```

Comment:
Maybe we could call this field as `NullableData` or `NullableCustomData` to reveal its intention.

Reply:
Sure. `NullableCustomData` is the most descriptive.